src/components/index.ts

```diff
@@ -17,10 +17,13 @@ export * from "./table";
 export * from "./xDragBoxLayer";
 export * from "./yDragBoxLayer";
 
-export class Alignment {
-  static TOP = "top";
-  static BOTTOM = "bottom";
-  static LEFT = "left";
-  static RIGHT = "right";
-  static CENTER = "center";
-}
+/**
+ * @deprecated just use string literals for alignment
```

Comment:
One thing about that that I really don't like is that the const name and the type name are the same string. Whenever I see one of these names, i have to think about whether or not it refers to the type or the object (which ISN'T EVEN OF THE TYPE).

"Don't make me think."

Maybe this?
```typescript
export const BarOrientationEnum = makeEnum(["vertical", "horizontal"]);
export type BarOrientation = keyof typeof BarOrientationEnum;
```

It'd also be cool to rename the function to just `enum` since that'd look pretty good:

```typescript
export const BarOrientationEnum = enum(["vertical", "horizontal"]);
export type BarOrientation = keyof typeof BarOrientationEnum;
```